### PR TITLE
Added switch for using local and remote versions

### PR DESCRIPTION
**Pull request by gavinwakeley1**
AB#8096
AB#8513

### CODE REVIEW

File: launch-or-update-local.ps1 [hunk 1]
```diff
@@ -24,28 +25,35 @@ param(
...
 {
-    write-host "Getting Latest Chart Versions..."
-    invoke-expression -Command "./update-chart-versions.ps1 -chart $chart -v development"
-}
-else
-{
-    write-host "Getting Master Chart Versions..."
-    invoke-expression -Command "./update-chart-versions.ps1 -chart $chart -v master"
-}
+    if (($latest -ne "false") -and ($l -eq $false))
```
Comment (stuart-bass-bjss) [inline launch-or-update-local.ps1:39]: Is it right to get the latest chart versions if update is false? won't this cause issues, as the version numbers will be updated, but the dependencies won't be downloaded?

Comment (gavinwakeley1) [reply launch-or-update-local.ps1:39]: I think that switch was mostly for doing master release updates (i.e. just the yaml file). I can remove it if you prefer and leave it as: useRemote? 
yes then always update, 
no then stay as you are. 

That way you're always pulling the latest repos from the acr if you're getting a remote branch

Comment (stuart-bass-bjss) [reply launch-or-update-local.ps1:39]: So, I think the cases we have are:
- Launch with latest versions from development (download if necessary)
- Launch with latest versions from master (download if necessary)
- Launch with the versions I currently have set up (don't download or update versions)
maybe we should just have a single flag to reflect these scenarios?

Comment (gavinwakeley1) [reply launch-or-update-local.ps1:39]: I've stripped out the update now and left it as master/development (both update) or a second option that stays as you are (no update). Is that ok? (not pushed it yet as I'm troubleshooting a 'hidden return char' bug

Comment (stuart-bass-bjss) [reply launch-or-update-local.ps1:39]: seems reasonable

Comment (gavinwakeley1) [reply launch-or-update-local.ps1:35]: ok, updated with changes and removed hidden chars from charts.yaml

**Pull request merged**